src/SkirtBrim.cpp

```diff
@@ -668,7 +668,7 @@ void SkirtBrim::generateSupportBrim()
 
         storage.support_brim.add(brim_line);
 
-        const coord_t length = skirt_brim_length + storage.support_brim.polygonLength();
+        const coord_t length = (adhesion_type == EPlatformAdhesion::NONE) ? skirt_brim_length : skirt_brim_length + storage.support_brim.polygonLength();
```

Comment:
I'll add a comment here -- I think otherwise you'd get future developers changing the portion between the `?` and `:` to `0`, because they forget about the support-brim 😁

Reply:
Oh yes. I thought I did made a comment there.. but apparently I didn't :/
Will do it (y)

Reply:
Never mind, it's literally in a 'support' function 😅

Reply:
... but the comment clears things up in any case!